include/zephyr/bluetooth/iso.h

```diff
@@ -217,6 +217,18 @@ struct bt_iso_recv_info {
 	uint8_t flags;
 };
 
+/** @brief ISO Meta Data structure for transmitted ISO packets. */
+struct bt_iso_tx_info {
+	/** CIG reference point or BIG anchor point of a transmitted SDU, in microseconds. */
+	uint32_t ts;
```

Comment:
```suggestion
	uint32_t timestamp;
```

Reply:
This PR introduces new fields, do not see a reason to have separate PR for the changes in this PR.

A separate PR can handle other fields that are upstream and follow the API deprecation rules (which will not be required for the new API in this PR).

Reply:
> If we are expanding this, the other fields called `ts` and `sn` should also be expanded, possibly also `sca` and `mse` by going with this logic.

`sca` and `mse` are BT Spec abbreviations, these do not need change.

Though `sn` is a BT Spec abbreviations, it means a 1-bit value in Link Layer Specification.

`to` was the trigger for my change request, as it could mean timeout.

Reply:
>  I rather have ts if it means that it's consistent with the existing API

`ts` is ok by me to remain as it. `sn` and `to` should change in this PR and any in #44117 introduced as new fields to avoid future deprecation.

Reply:
Renamed `sn` to `seq_num` in https://github.com/zephyrproject-rtos/zephyr/pull/44117, suggest to do the same here, and fix `ts` in a later PR

Reply:
Renamed `to` and `sn` as suggested